book/build-service.md

```diff
@@ -0,0 +1,142 @@
+# Build Service
+
+
+## Overview
+
+The Build Service is composed of controllers that create and configure build pipelines. The main input for the Build Service is a Component CR managed by the [Hybrid Application Service (HAS)](hybrid-application-service.md).
+
+![](../diagrams/build-service/build-service.drawio.svg)
+
+### Dependencies
+
+The Build Service is dependent on the following services:
+- [Pipeline Service](./pipeline-service.md)
+  - Pipeline execution, Pipeline logging
+- [Hybrid Application Service](./hybrid-application-service.md)
+  - Provides Component CR with annotations and `.status.devfile` which is used for PipelineRun configuration.
+- [Image Controller](./image-controller.md)
+  - Generation of container image repository which is used by PipelineRun
+
+## Controllers
+
+The Build Service contains these controllers:
```

Comment:
Where does the image controller land? Is that within the build service or outside it?

Reply:
Nevermind, I see that it is another top-level document (currently empty) so I assume that it falls outside of the build service.

Reply:
Added image controller into diagram

Reply:
Thanks for the image. Is there any intention to populate `image-controller.md` in this PR?

Reply:
Absolutely not.

Reply:
That's fine. Can you put some "content" into it as a TODO/nod to future work/intentionally left blank?

Reply:
I used same approach as it was done before:
https://github.com/redhat-appstudio/book/blob/main/book/build-service.md
https://github.com/redhat-appstudio/book/blob/main/book/hybrid-application-service.md
https://github.com/redhat-appstudio/book/blob/main/book/gitops-service.md
https://github.com/redhat-appstudio/book/blob/main/book/java-rebuilds-service.md
https://github.com/redhat-appstudio/book/blob/main/book/service-provider-integration.md
https://github.com/redhat-appstudio/book/blob/main/book/workspace-and-terminal-service.md